pkg/scheduling/volumeusage.go

```diff
@@ -126,22 +122,34 @@ func resolveDriver(ctx context.Context, kubeClient client.Client, volumeName str
 		if err != nil {
 			return "", err
 		}
-		if driverName != "" {
+		if driverName == "" {
```

Comment:
This seems incorrect. Unless I'm misunderstanding something

Reply:
Definitely a typo...fixed now